### PR TITLE
Replication

### PR DESCRIPTION
**Pull request by alainjobart**
@enisoc
Some clean up.

### CODE REVIEW

File: go/vt/wrangler/reparent.go [hunk 1]
```diff
@@ -72,54 +72,39 @@ func (wr *Wrangler) shardReplicationStatuses(ctx context.Context, shardInfo *top
...
-			rpcCtx.err = err
-			if err == nil {
-				rpcCtx.status = &myproto.ReplicationStatus{Position: pos}
-			}
+			wg.Add(1)
+			go func(i int, ti *topo.TabletInfo) {
+				defer wg.Done()
+				pos, err := wr.tmc.MasterPosition(ctx, ti)
+				if err != nil {
+					rec.RecordError(fmt.Errorf("MasterPosition(%v) failed: %v", ti.Alias, err))
```
Comment (enisoc) [inline go/vt/wrangler/reparent.go]: Shouldn't golint suggest a 'return' here, so the path below can be unindented?

File: go/vt/wrangler/reparent.go [hunk 2]
```diff
@@ -72,54 +72,39 @@ func (wr *Wrangler) shardReplicationStatuses(ctx context.Context, shardInfo *top
...
-		if tablet.Type == topo.TYPE_MASTER || tablet.IsSlaveType() {
 			wg.Add(1)
-			go f(i)
-		} else {
-			wr.logger.Infof("tabletReplicationPositions: skipping tablet %v type %v", tablet.Alias, tablet.Type)
+			go func(i int, ti *topo.TabletInfo) {
+				defer wg.Done()
+				status, err := wr.tmc.SlaveStatus(ctx, ti)
+				if err != nil {
+					rec.RecordError(fmt.Errorf("SlaveStatus(%v) failed: %v", ti.Alias, err))
```
Comment (enisoc) [inline go/vt/wrangler/reparent.go]: Same as above.

**Comment by enisoc** [comment]
LGTM

File: go/vt/wrangler/reparent.go (hunk 1 above)
Comment (alainjobart) [inline go/vt/wrangler/reparent.go]: golint didn't complain, strange. Fixed it anyway.

**Pull request merged**